src/renderer/models/account.ts

```diff
@@ -98,6 +98,7 @@ export class CGAccount implements IAccount {
             try {
                 return await keystore.decrypt(password);
             } catch (e) {
+                console.error(e);
```

Comment:
remove this :)

Reply:
but I had a hard time catching a bug because of it :D

Reply:
import and use electron-log :stuck_out_tongue:

Reply:
ah, didn't notice that one 👼